src/lib/components/Modal/Modal.jsx

```diff
@@ -97,9 +97,14 @@ export const Modal = ({
   };
 
   useEffect(() => {
+    const bodyElement = document.getElementsByTagName('body')[0];
+    bodyElement.style = 'overflow-y: hidden';
```

Comment:
I'm afraid the blocking feature needs to be smarter, at least it can cover the varying scrollbar width across OS's…

https://github.com/twbs/bootstrap/blob/main/js/src/util/scrollbar.js#L34